projects/packages/videopress/changelog/add-videopress-blocks-feedback-link

```diff
@@ -0,0 +1,4 @@
+Significance: minor
```

Comment:
I think the significance should be `patch`. That's why you had to fix the version issue.